src/Tizen.NUI/src/public/XamlBinding/BindableObject.cs

```diff
@@ -867,11 +868,13 @@ void ClearValue(BindableProperty property, bool fromStyle, bool checkAccess)
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        BindablePropertyContext CreateAndAddContext(BindableProperty property)
+        BindablePropertyContext CreateAndAddContext(BindableProperty property, bool isSetValueCase, bool valuePolicyApplied = false)
```

Comment:
parameter의 순서가 `isSetValueCase`는  `valuePolicyApplied`에 종속적이기 때문에 `valuePolicyApplied`가 먼저 오는것이 맞는것 같고 default parameter또한 둘 다 적용하는 것이 맞지 않을까요?

```suggestion
        BindablePropertyContext CreateAndAddContext(BindableProperty property, bool valuePolicyApplied = false, bool isSetValueCase = false)
```

Reply:
네. 말씀해주신 내용대로 수정하였습니다.